bin/test_build

```diff
@@ -3,7 +3,7 @@
 setup=aardvark,aardvark2,default,networking,switch
 faux=faux1,faux2
 faucet=faucet,gauge
-test=test_bacnet,test_hold,test_mudgee,test_brute,test_discover,usi,test_ipaddr
+modules=test_bacnet,test_hold,test_mudgee,test_brute,test_discover,usi,test_ipaddr
```

Comment:
modules is also too generic since something like "test_switch" are also modules. base? core?  How is this set of things different than the other set of things?